modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -45,6 +45,12 @@ For deploying {sno}, LVM Storage has the following limitations:
 ** It is possible to define the size of PE and LE during the physical and logical device creation.
 ** The default PE and LE size is 4 MB.
 ** If the size of the PE is increased, the maximum size of the LVM is determined by the kernel limits and your disk space.
+* To create volume snapshots and volume clones in multi-node topology, ensure that the following conditions are met:
+** The original persistent volume claim (PVC) is in `Bound` state.
+** The node must have additional storage after creating a volume snapshot and volume clone for further updating the original data source.
+* Creating volume snapshots and volume clones is based on the LVM thin pool capabilities.
+* Pods relying on the PVC that uses the snapshot data and clone data, volume snapshots, and volume clones can be created only on the node where the original data source is deployed.
+* If the PVC is not created before creating a volume snapshot, then the scheduler in multi-node OpenShift clusters does not provision the volume snapshots on the node where the original data source is deployed.
```

Comment:
@sr1kar99 For example, I tried rewriting this bullet (L53) to keep it simple and in active voice:

If the PVC is not present and available before creating a volume snapshot, then the scheduler in multi-node OpenShift clusters does not provision the volume snapshots on the node where you have deployed the original data source.

I am not sure if it is technically correct. You can check and if all is ok, you can follow this style or a similar style to rewrite.

Reply:
As confirmed by @radeore, this info also applies to volume clones. Hence, updated the line as follows:

If the PVC is not present and available before creating a volume snapshot or volume clone, then the scheduler in multi-node OpenShift clusters does not provision the volume snapshots and volume clones on the node where you have deployed the original data source.